### ISSUE TITLE
"lotus state compute-state --html" Broken after fvm upgrade

### ISSUE THREAD

**Issue by geoff-vball** [filecoin-project/lotus#8636]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [X] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
Daemon:  1.15.2-dev+interopnet+git.0295d2274.dirty+api1.5.0
Local: lotus version 1.15.2-dev+interopnet+git.0295d2274.dirty
```

### Describe the Bug

When running `lotus state compute-state --html`, the output contains the error `ERROR: template: message:3:24: executing "message" at <.Msg.To>: nil pointer evaluating *types.Message.To`.

This error only occurs when looking at the state **after** the FVM upgrade (3000 on interopnet).

### Logging Information

```text
.
```

### Repo Steps

lotus state compute-state --html

**Comment by jennijuju** [comment]
@arajasek given what steb said - should we expect the issue shouldn't exist when setting `LOTUS_VM_ENABLE_GAS_TRACING_VERY_SLOW` to be 1?

**Comment by arajasek** [comment]
@jennijuju No, we expect the issue to exist in all cases (before this fix) -- the problem is that we were only _trying_ to set the execution trace if we had an FVM execution trace (which we did when we had `LOTUS_VM_ENABLE_GAS_TRACING_VERY_SLOW=1`). 

So in the `LOTUS_VM_ENABLE_GAS_TRACING_VERY_SLOW=0` case, we would basically just set the execution trace to be nil (because `et` would be nil on this line https://github.com/filecoin-project/lotus/pull/8706/files#diff-9c51c6db9b9f6bea9717d162a7f80db6e0e144032849d83281750e3226d17ccaL359).
